doc/tools/coqrst/coqdomain.py

```diff
@@ -210,6 +215,15 @@ def _add_target(self, signode, name):
             signode['names'].append(name)
             signode['first'] = (not self.names)
             self._record_name(name, targetid, signode)
+        else:
+            # todo: make the following a real error or warning
+            names_in_subdomain = self.subdomain_data()
+            if name in names_in_subdomain:
+                try:
+                    print("Duplicate", self.subdomain, "name: ", name)
+                except: # UnicodeEncodeError in CI
+                    print("*** Exception")
+            # self._warn_if_duplicate_name(names_in_subdomain, name, signode)
```

Comment:
This should be a proper warning rather than a message, right?

Reply:
No, not yet.  If it's a warning, we have to fix the problems immediately, which we don't want to do.

Reply:
In particular, it will become much more reasonable to attack the fixing of these warnings once the SSR chapter syntax fixing pass is over (#13019). See also my previous comment at https://github.com/coq/coq/pull/12182#issuecomment-619619265.

Reply:
Sounds good.  I'm not sure where the unicodeerror comes from, but can we catch just that instead of the naked except?

Reply:
This explains where the error comes from: https://github.com/coq/coq/pull/13564#issuecomment-739125457.  I'm not going to fix that right now, but we should consider doing it.  Hence I catch the exception instead.

> can we catch just that instead of the naked except?

I guess.  This bit of code will go away soon.  It feels like we're spending more time than we really need to perfect what is supposed to be a simple fix.  "Not everything worth doing is worth doing well."  If these were changes to the kernel, I would definitely care a lot more about perfecting all the details.